tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -4327,6 +4327,12 @@ vxlan/test_vxlan_ecmp_switchover.py:
     conditions:
       - "(is_multi_asic == True) or (platform not in ['x86_64-8102_64h_o-r0', 'x86_64-8101_32fh_o-r0', 'x86_64-mlnx_msn4600c-r0', 'x86_64-mlnx_msn2700-r0', 'x86_64-mlnx_msn2700a1-r0', 'x86_64-kvm_x86_64-r0', 'x86_64-mlnx_msn4700-r0', 'x86_64-nvidia_sn4280-r0', 'x86_64-8102_28fh_dpu_o-r0'])"
 
+vxlan/test_vxlan_ecmp_vnet_ping.py:
+  skip:
+    reason: "Skip for Fairwater AI backend, as this feature is not been used"
+    conditions:
+      - "platform in ['x86_64-nvidia_sn5640-r0']"
```

Comment:
I think we should skip this based on topology, not platform.

Reply:
updated. Checked t1-isolated-d56u1-lag is our only topo with this test